tests/tracking/test_client.py

```diff
@@ -252,3 +253,63 @@ def test_registry_uri_from_implicit_tracking_uri():
         get_tracking_uri_mock.return_value = tracking_uri
         client = MlflowClient()
         assert client._registry_uri == tracking_uri
+
+
+def test_create_model_version_normal(mock_registry_store):
+    run_id = 'runid'
+    client = MlflowClient(tracking_uri='http://10.123.1231.11')
+    mock_registry_store.create_model_version.return_value = \
+        ModelVersion('name', 1, 0, 1, source='source', run_id=run_id)
+    model_version = client.create_model_version('name', 'source', 'runid')
+    assert(model_version.name == 'name')
+    assert(model_version.source == 'source')
+    assert(model_version.run_id == 'runid')
+    # verify that the store was not provided a run link
+    mock_registry_store.create_model_version.assert_called_once_with(
+        "name", 'source', 'runid', [], None)
+
+
+def test_create_model_version_run_link_in_notebook_with_default_profile(mock_registry_store):
+    experiment_id = 'test-exp-id'
+    hostname = 'https://workspace.databricks.com/'
+    workspace_id = '10002'
+    run_id = 'runid'
+    workspace_url = construct_run_url(hostname, experiment_id, run_id, workspace_id)
+    get_run_mock = mock.MagicMock()
+    get_run_mock.return_value = Run(RunInfo(run_id, experiment_id, 'userid', 'status', 0, 1, None),
+                                    None)
+    with mock.patch('mlflow.tracking.client.is_in_databricks_notebook', return_value=True), mock\
+            .patch('mlflow.tracking.client.get_workspace_info_from_dbutils',
+                   return_value=(hostname, workspace_id)):
```

Comment:
```suggestion
    with mock.patch('mlflow.tracking.client.is_in_databricks_notebook', return_value=True), \
            mock.patch('mlflow.tracking.client.get_workspace_info_from_dbutils',
                       return_value=(hostname, workspace_id)):
```

Reply:
Linters hate me today lol so I tried to do this and it complained about the continuation line being over-indented. Moving the second `mock` line back *1 space* has apparently fixed the problem and looks more readable than the original (though less clean than your suggestion)